src/components/button/styled.tsx

```diff
@@ -0,0 +1,266 @@
+          width: ${label ? "128px" : "40px"};
+          height: 40px;
+          font-family: IBM Plex Sans;
+          font-style: normal;
+          font-weight: bold;
+          font-size: 12px;
+          color: ${getColor(["green", "greenHaze"])};
+          &:hover {
+            border-style: solid;
+            border-color: ${getColor(["green", "hoverGreen"])};
+            border-width: 3px;
+            width: ${label ? "132px" : "44px"};
+            height: 44px;
+          }
+          &:active {
+            background: ${getColor(["green", "malachite"])};
+            border-style: solid;
+            border-color: ${getColor(["green", "malachite"])};
+            border-width: 3px;
+            width: ${label ? "132px" : "44px"};
+            height: 44px;
+            color: ${getColor(["white", "pure"])};
+          }
+          &:focus {
+            outline: none;
+          }
+          display: ${icon.props.name || isLoading ? "flex" : "block"};
+          flex-flow: row nowrap;
+          align-items: center;
+          .rmwc-icon {
+            margin-right: ${(label && icon.props.name) || isLoading ? "8px" : "0px"};
+            height: 24px;
+            width: 24px;
+          }
+          &.rmwc-icon path {
+              fill: ${getColor(["green", "malachite"])};
+          }
+        }
+        `,
+  ],
+  [
+    "default",
+    (label: string, icon: any, isLoading: boolean) => css`
+      background: ${getColor(["green", "greenHaze"])};
+      border-radius: 3px;
+      border-width: 0px;
+      width: ${label ? "128px" : "40px"};
+      height: 40px;
+      font-family: IBM Plex Sans;
+      font-weight: bold;
+      font-size: 12px;
+      color: ${getColor(["white", "pure"])};
+      &:hover {
+        border-style: solid;
+        border-color: ${getColor(["green", "hoverGreen"])};
+        border-width: 3px;
+        width: ${label ? "134px" : "46px"};
+        height: 46px;
+      }
+      &:active {
+        background: ${getColor(["green", "malachite"])};
+        border-style: solid;
+        border-color: ${getColor(["green", "malachite"])};
+        border-width: 3px;
+        width: ${label ? "134px" : "46px"};
+        height: 46px;
+      }
+      &:focus {
+        outline: none;
+      }
+      display: ${icon.props.name || isLoading ? "flex" : "block"};
+      flex-flow: row nowrap;
+      align-items: center;
+      .rmwc-icon {
+        margin-right: ${(label && icon.props.name) || isLoading ? "8px" : "0px"};
+        height: 24px;
+        width: 24px;
+      }
+    `,
+  ],
+])
+
+export const StyledButton = styled(({ label, icon, ...otherProps }) => (
+  <Button label={label} icon={icon} {...otherProps} />
+))`
+  ${props => {
+    if (props.disabled) {
+      const template = buttonTemplates.get("disabled")
+      return template ? template(props.label, props.icon, props.isLoading) : css``
+    }
+    const template = buttonTemplates.get(props.type)
```

Comment:
here could not be prop, so fallback to default should be implemented. Better to create function to access data in `Map`

Reply:
So the next line is:
`return template ? template(props.label, props.icon, props.isLoading) : css```
which returns a "default" which I want it to be empty CSS. Is this what you mean?